src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/model/stats/Direction.groovy

```diff
@@ -0,0 +1,13 @@
+package org.openkilda.functionaltests.model.stats
+
+enum Direction {
+    FORWARD("forward"),
+    REVERSE("reverse"),
+    BOTH("*")
```

Comment:
probably not important, but I'm just curious why a start and not a word "both" as in the other enum values?